components/config-mgmt-service/grpcserver/node_export.go

```diff
@@ -117,14 +117,10 @@ func (s *CfgMgmtServer) exportNodes(ctx context.Context, request *pRequest.NodeE
 	// even after the node no longer exists
 	nodeFilters["exists"] = []string{"true"}
 
-	if sortField == "" {
-		sortField = backend.CheckIn
-	} else {
-		sortField = params.ConvertParamToNodeRunBackend(sortField)
-	}
```

Comment:
This code was misleading because:

* `sortField` will never be blank by the time the code gets here, so the first branch is not necessary.
* `sortField` has already been passed through `ConvertParamToNodeRunBackend` in the call to `request.Sorting.GetParameters()` a few lines up.